.github/workflows/build.yml

```diff
@@ -0,0 +1,63 @@
+on:
+  push:
+    branches:
+      - master
+
+name: Wally Build
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
```

Comment:
You should consider using an "older" version of Linux than Ubuntu. Wally is advertised to work with RHEL and Debian, and they'll get linker errors if built with a version of glibc that's too new. 

For my PR a while ago I used CentOS 8 which should be good for a long while https://github.com/zsa/wally/pull/124

Reply:
GitHub CI [doesn't support non-Ubuntu linux runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources) (it's possible with self-hosted runners though), otherwise I probably would've used something besides Ubuntu. I agree that it would be better to use something besides Ubuntu though, just a limitation of the available runners. 

I suppose as an alternative we could just use the Ubuntu runner to build Wally in your CentOS 8 container and extract the executable that way. I'll try to refactor to that approach instead.

Reply:
Aw man what a shame. I wonder why they make this limitation.

I did a bit of looking around and it seems like docker in docker is not supported: https://github.community/t/is-docker-in-docker-going-to-be-available-soon/130533

However it does look like there's a PPA with older GLIBC support: https://launchpad.net/~ubuntu-toolchain-r/+archive/ubuntu/ppa/+build/14520669 whether we're supposed to use it in much newer Ubuntu or not is uncertain. But this looks like an "official" PPA, and it's a compatible GLIBC version that CentOS 8 uses. 

```
[root@2f90a8190269 /]# ldd --version
ldd (GNU libc) 2.28
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.
[root@2f90a8190269 /]# cat /etc/os-release
NAME="CentOS Linux"
VERSION="8"
```